### PR TITLE
Fix LinkedIn profile field not updating

### PR DESCRIPTION
**Pull request by tudor-gala**

### CODE REVIEW

File: app/Http/Controllers/PeopleController.php [hunk 1]
```diff
@@ -205,6 +205,12 @@ public function update(Request $request, $contactId)
             $contact->twitter_profile_url = null;
         }
 
+        if ($request->input('linkedin') != '') {
```
Comment (Phrohdoh) [inline app/Http/Controllers/PeopleController.php]: Should probably `trim(..)` this before checking against an empty string.

Comment (tudor-gala) [inline app/Http/Controllers/PeopleController.php]: For the sake of consistency I just followed the exact same pattern that the other fields had, as I didn't want to change more lines than needed, should I just trim() all the other field inputs as well?

Comment (Phrohdoh) [inline app/Http/Controllers/PeopleController.php]: Will defer to @djaiss then. I hadn't taken the time to look at the other fields, sorry about that.

My opinion is yes trim all in another commit.

**Comment by djaiss** [comment]
Thanks a lot. This is trickier than this actually.

I've just forgotten to remove the field from the Edit personal information view.

LinkedIn is now associated to the Work information section. To update it, you need to edit the Work information - not the Personal information.

I've just pushed a fix in production to change this - LinkedIn is now completely out of the Personal information section.

In my opinion, the Personal information shouldn't list LinkedIn.

That being said, thanks a lot for caring and submitting this PR. I truly appreciate it.

**Pull request closed without merging**